app/assets/stylesheets/user_groups/_edit.css.scss

```diff
@@ -0,0 +1,3 @@
+.user-groups-edit {
+    .user_group_user_ids { margin-bottom: 1rem; }
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `user_group_user_ids` should be written in lowercase with hyphens